contracts/dezswap_pair/src/contract.rs

```diff
@@ -276,12 +278,34 @@ pub fn provide_liquidity(
         // Initial share = collateral amount
         let deposit0: Uint256 = deposits[0].into();
         let deposit1: Uint256 = deposits[1].into();
-        match (Decimal256::from_ratio(deposit0.mul(deposit1), 1u8).sqrt() * Uint256::from(1u8))
-            .try_into()
+        let share: Uint128 = match (Decimal256::from_ratio(deposit0.mul(deposit1), 1u8).sqrt()
+            * Uint256::from(1u8))
+        .try_into()
         {
             Ok(share) => share,
             Err(e) => return Err(ContractError::ConversionOverflowError(e)),
-        }
+        };
+
+        // MINIMUM_LIQUIDITY_AMOUNT is for the contract's stake
+        messages.push(CosmosMsg::Wasm(WasmMsg::Execute {
+            contract_addr: deps
```

Comment:
pair_info.liquidity_token.to_string()

Reply:
Isn't it for use `liquidity0000` mock address in testing